File: function/5.py
input_id = input("Enter the ID please.\n")

def login(_id):
	members = ['ansxodbs', 'rlagkdns', 'duagkswn']
	for member in members:
		if member == _id:
			return True
	return False

if  login(input_id):
	print("Hello, " + input_id)
else:
	print("Who are you?")

File: function/5.rb
puts("enter the ID.")
input_id = gets.chomp()

def login(_id)
	members = ['ansxodbs', 'k8805', 'showme']
	for member in members do
		if member == _id
			return true
		end
	end
	return false
end

if login(input_id)
	puts('Hello, ' + input_id)
else
	puts("Who are you?")
end
